pkgs/native_toolchain_c/lib/src/cbuilder/cbuilder.dart

```diff
@@ -46,6 +46,15 @@ class CBuilder extends CTool implements Builder {
   /// Defaults to `true`.
   final bool ndebugDefine;
 
+  /// Libraries to dynamically link to.
+  ///
+  /// The libraries are expected to be at the root of the output directory of
+  /// the build hook invocation.
+  ///
+  /// When using this option ensure that the builders producing the libraries
+  /// to link to are run before this builder.
+  final List<String> dynamicallyLinkTo;
```

Comment:
Are these strings names or file paths? Please document. Also, why prefer names over paths or vice versa? (Should probably be documented in the PR description as design choice.)